src/odemis/driver/simcam.py

```diff
@@ -258,8 +265,8 @@ def _generate(self):
             gen_img = self._write_txt_image(gen_img, txt)
 
         # update fake output metadata
-        exp = timer.period
-        metadata[model.MD_ACQ_DATE] = time.time() - exp
+        self._last_acq_time = time.time() - exp
```

Comment:
I thought the odemis-style was to define the ACQ_DATE at the start of the acquisition. Now it is done more or less at the end with some steps in between.
This could be prone to errors when the code is changed in later (e.g. acquisitions taken in the past because the sleep is not sufficient.
I don't think it is a big issue, unless it doesn't comply with the general odemis-style but that is up to you @pieleric.

Reply:
Note that this commit doesn't change the current behaviour. 
In any case, the convention is that the ACQ_DATE indicates the time at the beginning of the acquisition. That's the case here, as it reads the time at the end and subtracts the exposure time. The reason that it's done this way is that in the simulator the exposure time is simulated by using the `RepeatingTimer` which already takes care of sleeping so it'd be difficult to store the time before the sleep.